src/electionguard/facades/bignum4096.cpp

```diff
@@ -188,6 +187,9 @@ namespace electionguard::facades
     }
     uint64_t Bignum4096::lessThan(uint64_t *a, uint64_t *b)
     {
+        if (use32BitMath) {
```

Comment:
Does this mean that we are not using the 32bit version of hacl?  On the C++ side we were using the 32bit version of the Bignum4096 to speed things up.  Just checking if this is still the case.